### PR TITLE
Update the description for DataGrid - Adaptability Overview

### PR DESCRIPTION
**Pull request by AlexJameson**
https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/GridAdaptabilityOverview/jQuery/Light/

### CODE REVIEW

File: JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md [hunk 1]
```diff
@@ -1 +1,5 @@
-The **DataGrid** can adapt its layout to narrow screens. If screen width is not enough to display all columns, the widget hides them one by one. To view values of hidden columns, click the ellipsis button in the last column of the required row. Switch between the landscape and portrait layouts to see how the **DataGrid** looks on wide and narrow screens.
\ No newline at end of file
+**DataGrid** automatically adapts its layout to screens of different widths. Use buttons in the top right corner to switch between the mobile and tablet screens. If columns do not fit the chosen screen type, the widget hides them one by one from right to left.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md]:
```suggestion
**DataGrid** automatically adapts its layout to screens of different widths. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the chosen orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.
```

File: JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md [hunk 2]
```diff
@@ -1 +1,5 @@
-The **DataGrid** can adapt its layout to narrow screens. If screen width is not enough to display all columns, the widget hides them one by one. To view values of hidden columns, click the ellipsis button in the last column of the required row. Switch between the landscape and portrait layouts to see how the **DataGrid** looks on wide and narrow screens.
\ No newline at end of file
+**DataGrid** automatically adapts its layout to screens of different widths. Use buttons in the top right corner to switch between the mobile and tablet screens. If columns do not fit the chosen screen type, the widget hides them one by one from right to left. 
+
+To enable this behavior in your application, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true**.
+
+**DataGrid** displays hidden columns' data in the adaptive row as [Form](/Documentation/Guide/Widgets/Form/Overview/) items. You can use the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onAdaptiveDetailRowPreparing) handler to access this data. Refer to the handler's description to see the limitations.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md]: The paragraph looks unnecessary. This handler does allow you to customize the adaptive detail row, but it seems that only few customers really need that. And the demo doesn't show this handler.

> Refer to the handler's description to see the limitations.

I think we mention limitations too often, it makes us look bad. We shouldn't do that unless the limitations are serious and easy to encounter.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — DISMISSED** [review]

**Review by Abadzhev — COMMENTED** [review]
Looks good to me. Thanks.

File: JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md [hunk 3]
```diff
@@ -1 +1,3 @@
-The **DataGrid** can adapt its layout to narrow screens. If screen width is not enough to display all columns, the widget hides them one by one. To view values of hidden columns, click the ellipsis button in the last column of the required row. Switch between the landscape and portrait layouts to see how the **DataGrid** looks on wide and narrow screens.
\ No newline at end of file
+**DataGrid** automatically adapts its layout to screens of different widths. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the chosen orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md]:
```suggestion
**DataGrid** automatically adapts its layout to screens with different sizes. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the selected orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.
```

File: JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md [hunk 4]
```diff
@@ -1 +1,3 @@
...
+
+To enable this behavior in your application, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true**.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md]:
```suggestion
To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true**.
```

**Review by dirk-pieterse — DISMISSED** [review]

File: JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md [hunk 5]
```diff
@@ -1 +1,3 @@
-The **DataGrid** can adapt its layout to narrow screens. If screen width is not enough to display all columns, the widget hides them one by one. To view values of hidden columns, click the ellipsis button in the last column of the required row. Switch between the landscape and portrait layouts to see how the **DataGrid** looks on wide and narrow screens.
\ No newline at end of file
+**DataGrid** automatically adapts its layout to screens with different sizes. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the selected orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.
```
Comment (pavel-2020) [inline JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md:1]: "Information from the hidden columns is still available in **adaptive detail rows**."
This sentence left me wondering what these 'adaptive detail rows' are.  The sentence from the previous version of  the demo description explains this concept better: "To view values of hidden columns, click the ellipsis button in the last column of the required row.", doesn't it?

**Review by pavel-2020 — APPROVED** [review]

**Pull request merged**